bdb/bdb_verify.c

```diff
@@ -915,15 +915,16 @@ static int bdb_verify_ll(
                 }
 
             } else if (bdb_state->ixcollattr[ix]) {
-                if (dbt_data.size !=
-                    (sizeof(unsigned long long) + bdb_state->ixcollattr[ix])) {
+                if (dbt_data.size != (sizeof(unsigned long long) +
+                                      4 * bdb_state->ixcollattr[ix])) {
```

Comment:
See [here](https://github.com/bloomberg/comdb2/blame/master/db/tag.c#L7459). This was a verify day1 bug for decimal indexes? But I think it would be better to use a marco or sizeof(short).. But this is out of this PR's scope. Fixed this to make verify pass.